### PR TITLE
Security example for app.keys

### PR DESCRIPTION
**Pull request by nbeguier**
**Documentation**:
  - Improve `app.keys` definition
  - Add a security example for `app.keys`

### CODE REVIEW

File: docs/api/index.md [hunk 1]
```diff
@@ -187,8 +187,8 @@ https.createServer(app.callback()).listen(3001);
  example the following are acceptable:
 
 ```js
-app.keys = ['im a newer secret', 'i like turtle'];
-app.keys = new KeyGrip(['im a newer secret', 'i like turtle'], 'sha256');
+app.keys = ['insert 64 bytes random string', 'insert another 64 bytes random string']; // sha1 is less secured
+app.keys = new KeyGrip(['insert 64 bytes random string', 'insert another 64 bytes random string'], 'sha256');
```
Comment (marswong) [inline docs/api/index.md]: using truely 64-byte strings makes more sense here :)

Comment (3imed-jaberi) [inline docs/api/index.md]:
```suggestion
app.keys = new KeyGrip(['insert 64 bytes random string  like im a newer secret', 'insert another 64 bytes random string  like i like turtle'], 'sha256');
```

File: docs/api/index.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ app.listen(3000);
   the following are supported:
 
   - `app.env` defaulting to the __NODE_ENV__ or "development"
-  - `app.keys` array of signed cookie keys
+  - `app.keys` `KeyGrip` instance, or array of signed cookie keys (less secured)
```
Comment (jonathanong) [inline docs/api/index.md:115]: secure*

**Review by anlexN — APPROVED** [review]
i think it is ok, please merge.

**Pull request closed without merging**